applications/asset_tracker_v2/src/modules/Kconfig.gnss_module

```diff
@@ -48,6 +48,37 @@ config GNSS_MODULE_ELEVATION_MASK
 	default NRF_CLOUD_AGPS_ELEVATION_MASK
 	depends on NRF_CLOUD_AGPS_FILTERED
 
+choice GNSS_MODULE_DYNAMICS_MODE
+	prompt "Tune GNSS performance for a specific use case"
```

Comment:
is this really the most optimal title possible ?
Combined with the config choices below, this doesn't play well in `menuconfig`.

Could you try to shorten the text, while still keep the title informative enough.

Remember users can open help text if the need more detsiled information, so no need to write on details in the title field.
![image](https://user-images.githubusercontent.com/9802583/190104013-ec509b24-a807-499d-aed1-0823ae7680f5.png)